cmd/server/usersrv/usersrv.go

```diff
@@ -35,6 +36,7 @@ func (t *TokenIssuer) IssueToken(userID string, exp time.Time) (string, string,
 		IssuedAt(now).
 		NotBefore(now.Add(-time.Minute)).
 		JwtID(id).
+		Claim("email", email).
```

Comment:
Same note here, let's use `emails` and the full `[]string`

Reply:
SG